main.cpp

```diff
@@ -6,51 +6,46 @@ using std::cout;
 using std::endl;
 using std::string;
 
-// These are empty methods for use case to be able to build and run.
-void ConnectWithTimeout(const std::string& hostname,
-                        const std::string& protocol, int timeout_ms) {
-  (void) hostname;
-  (void) protocol;
-  (void) timeout_ms;
-}
-
-void SetSynchronizedFiles(const std::vector<char*>& synchronized_paths) {
-  (void) synchronized_paths;
-}
-
 // Create variables to hold values of parsed flags.
 // Consider wrapping those in 'struct { ... } flags;' to cleanly
 // separate them from usual variables.
-args::String hostname;
-args::Enum protocol;
-args::Int timeout_ms;
+args::Int size;
 args::Bool verbose;
 
 int main(int argc, char** argv) {
   // Bind our variables to new flags.
-  args::AddString(&hostname, "hostname", args::REQUIRED,
-                  "Hostname of the synchronization server.");
-  args::AddEnum(&protocol, {"protocol", 'p'}, args::REQUIRED,
-                "Synchronization protocol to use.",
-                {"http", "ftp", "https"});
-  args::AddInt(&timeout_ms, 't', args::OPTIONAL,
-               "Connection timeout in ms. Default value is 1000.",
-               0, 60000);
-  args::AddBool(&verbose, {"verbose", 'v'}, args::OPTIONAL,
-                "Enable verbose logging?");
+  args::AddBool(&verbose, {'v', "verbose"}, args::OPTIONAL,
+                "Enable verbose logging");
+
+  args::AddInt(&size, {'s', "size"}, args::OPTIONAL,
+               "Size of something");
 
   // Parse command line flags. Die on error.
   args::Parse(&argc, &argv);
-
+  
+  if (verbose.present()){
```

Comment:
Also, the condition should read "verbose.present() && verbose.get()" to guard against --verbose=false.